config/helm/aws-node-termination-handler/README.md

```diff
@@ -69,7 +69,8 @@ Parameter | Description | Default
 `webhookTemplateConfigMapKey` | Name of the template file stored in the configmap| None
 `metadataTries` | The number of times to try requesting metadata. If you would like 2 retries, set metadata-tries to 3. | `3`
 `cordonOnly` | If true, nodes will be cordoned but not drained when an interruption event occurs. | `false`
-`taintNode` | If true, nodes will be tainted when an interruption event occurs. Currently used taint keys are `aws-node-termination-handler/scheduled-maintenance`, `aws-node-termination-handler/spot-itn`, and `aws-node-termination-handler/asg-lifecycle-termination` | `false`
+`drainOnRebalance` | If true, nodes will be drained when a rebalance recommendation notice is received. | `false`
```

Comment:
For consistency, could you re-name to something like `enableRebalanceDraining` and add this config to **IMDS Mode Configuration** section? Around line 103 in this file

Reply:
Oh, sure. I followed the `cordonOnly` implementation but `enableRebalanceDraining` is way more consistent with what we already have.